android/app/src/main/res/values-fr/strings.xml

```diff
@@ -638,15 +638,11 @@
 	<string name="bookmark_list_description_hint">Tapez une description (texte ou html)</string>
 	<string name="not_shared">Privé</string>
 	<string name="speedcams_alert_title">Radars de vitesse</string>
-	<!-- Generic «Always» string -->
-	<string name="always">Toujours</string>
-	<!-- Generic «Never» string -->
-	<string name="never">Jamais</string>
 	<string name="place_description_title">Description d\'endroit</string>
 	<!-- this text will be shown in application notification preferences opposite checkbox which enable/disable downloader notifications. Devices on Android 8+ are affected. -->
 	<string name="notification_channel_downloader">Téléchargeur carte</string>
-	<!-- Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit -->
-	<string name="pref_tts_speedcams_auto">Alerte sonore en cas de risque de dépassement de la limite de vitesse</string>
+	<!-- "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit -->
+	<string name="pref_tts_speedcams_auto">Pour t\'avertir d\'un excès de vitesse</string>
```

Comment:
Please remove the “Pour t’” — it’s not a title, it’s a command name.

Reply:
@fitojb should it be `'Avertir d'un excès de vitesse`?

Reply:
Modified...

![ios_speed_cam_prefs_fr1](https://github.com/organicmaps/organicmaps/assets/25888296/e24d33ce-15bb-4ef6-a7e2-8661a48b0da0)